tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -348,12 +396,15 @@ Replace data files for amendment
     user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}dates.csv
     user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}dates.meta.csv
     user clicks button  Upload data files
-    user checks headed table body row cell contains  Status          2  Complete   wait=180
+    user opens details dropdown  Footnote 2 seven filters
+    user clicks button  Delete footnote
+    user clicks button  Confirm
+    #EES-1442: Bug when Confirm data replacement button doesn't show
+    user reloads page
+    user waits until button is enabled  Confirm data replacement
 
 Confirm data replacement
     [Tags]  HappyPath
-    user waits until page contains  Data blocks: OK
```

Comment:
I'd keep these checks of `Data blocks: OK` and `Footnotes: OK` in the test suite. They could be included after the Delete footnote/Confirm lines, but before reloading the page.

I would also include a additional checks that they're correct before deleting the footnote too.